.env.example

```diff
@@ -1,3 +1,5 @@
 APP_PORT=4000
+NODE_TLS_REJECT_UNAUTHORIZED = '0'
 NASA_API_KEY=your_nasa_token
 NASA_API_NEO_URL=https://api.nasa.gov/neo/rest/v1/feed
+NASA_API_ROVER_URL=https://api.nasa.gov/mars-photos/api/v1/rovers/curiosity/latest_photos
```

Comment:
let's leave only this part NASA_API_ROVER_URL=https://api.nasa.gov

Reply:
@Mikhail1992 done